### ISSUE TITLE
Accomplishments only collect basic data

### ISSUE THREAD

**Issue by leonardiwagner** [leonardiwagner/scrapedin#1]
- It's only getting the basic data before clicking on 'show' more
- Is also known that the accomplishment items can be lower than actual items if there are too many items LinkedIn doesn't show all before clicking on 'show more'

**Comment by DrakenWan** [comment]
I am making a chrome extension to help the recruiters get linkedin profile data. I have so far been able to successfully get the education and experience section. I would like to point out that in my attempt to scrape the data the only way you can get those data is if the "show more" anchors are clicked on. 
My extension actually strives to automatically fill in the text fields of the "recruitment form". So I have directed the user to scroll to the section, click on the anchor "show more" and then scroll a little bit (scroll event capture) and things would work out just fine.
But I feel this is not a proper solution for it.
You can check out the chrome extension [here](https://github.com/DrakenWan/Rekrut).

**Comment by leonardiwagner** [comment]
@DrakenWan what do you do if a user has 30 items (since LinkedIn only load 10 items per "show more" click)? Do you click, wait, see if there are more items to load and click it again and do that multiple times?

**Comment by DrakenWan** [comment]
Yeah, I've got to do that multiple times and then scroll to trigger the extraction function. I don't like this thing but due to limitations of chrome extension APIs I have done this.
What I hate is that I cannot even import libraries to make scraping easier for me.

**Comment by leonardiwagner** [comment]
Alright, so we are going to do that on scrapedin as well, thank you for your information @DrakenWan
